src/policy/fees_args.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_POLICY_FEES_ARGS_H
+#define BITCOIN_POLICY_FEES_ARGS_H
+
+#include <fs.h>
+
+class ArgsManager;
+
+fs::path FeeestPath(const ArgsManager& argsman);
```

Comment:
Please add a doc comment for this function. E.g.
```
/** Return path of fee estimates data file. */
```

Reply:
Fixed as of b96e009a8a36f4feeee49d1a3c8f85fb9ab0d920